index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My Portfolio</title>
+    <title>Portfolio Home Page</title>
```

Comment:
Great job team 🤝

- [ ] I would suggest you use something more specific for your title such as `Kaizi Paul Portfolio`.